android/build.gradle

```diff
@@ -34,6 +34,8 @@ allprojects {
             url("$rootDir/../node_modules/jsc-android/dist")
         }
 
+        maven { url 'https://maven.google.com' }
```

Comment:
Why is this needed?

Reply:
@joydeep33 It's required for image picker. I have used this [image picker](https://www.npmjs.com/package/react-native-image-crop-picker). in the installation part it was written to add this.